server/src/main/java/org/apache/druid/server/coordinator/CoordinatorDynamicConfig.java

```diff
@@ -299,7 +314,8 @@ public boolean equals(Object o)
 
     CoordinatorDynamicConfig that = (CoordinatorDynamicConfig) o;
 
-    if (millisToWaitBeforeDeleting != that.millisToWaitBeforeDeleting) {
+    if (millisLagSinceCoordinatorBecomesLeaderBeforeCanMarkAsUnusedOvershadowedSegments !=
+        that.millisLagSinceCoordinatorBecomesLeaderBeforeCanMarkAsUnusedOvershadowedSegments) {
```

Comment:
Whoa, it takes me a deal of concentration just to check those two variables represent the same property. Now I know when the bigger is not better regarding the variable naming.

Reply:
Do you have some better names in mind?

Reply:
`millisLagBeforeMarkingUnusedOvershadowed`

Reply:
This property is already renamed to `leadingTimeMillisBeforeCanMarkAsUnusedOvershadowedSegments` in #7653.